src/extension/chatSessions/vscode-node/chatSessions.ts

```diff
@@ -136,12 +142,52 @@ export class ChatSessionsContrib extends Disposable implements IExtensionContrib
 					copilotSessionsProvider.openSessionsInBrowser(chatSessionItem);
 				})
 			);
+			this.copilotCloudRegistrations.add(
+				vscode.commands.registerCommand('github.copilot.cloud.sessions.proxy.checkoutFromDescription', async (path: string | undefined) => {
+					await this.installPullRequestExtension();
+					await vscode.commands.executeCommand('pr.checkoutFromDescription', { path });
```

Comment:
I am not seeing these commands execute on my machine and I don't know why (@rebornix)